app/experimenter/experiments/tests/api/v4/test_serializers.py

```diff
@@ -0,0 +1,83 @@
+import datetime
+import json
+import os
+from jsonschema import validate
+
+from django.test import TestCase
+
+from experimenter.experiments.models import Experiment
+from experimenter.experiments.tests.factories import (
+    ExperimentFactory,
+    ExperimentVariantFactory,
+)
+from experimenter.experiments.api.v4.serializers import ExperimentRapidRecipeSerializer
+
+
+class TestExperimentRapidRecipeSerializer(TestCase):
+    def test_serializer_outputs_expected_schema(self):
+        audience = Experiment.RAPID_AUDIENCE_CHOICES[0][1]
+        features = [feature[0] for feature in Experiment.RAPID_FEATURE_CHOICES]
+        normandy_slug = "experimenter-normandy-slug"
+        today = datetime.datetime.today()
+        experiment = ExperimentFactory.create(
+            type=Experiment.TYPE_RAPID,
+            rapid_type=Experiment.RAPID_AA_CFR,
+            audience=audience,
+            features=features,
+            normandy_slug=normandy_slug,
+            proposed_enrollment=9,
+            proposed_start_date=today,
+        )
+
+        ExperimentVariantFactory.create(
+            experiment=experiment, slug="control", is_control=True
+        )
+        ExperimentVariantFactory.create(experiment=experiment, slug="variant-2")
+
+        serializer = ExperimentRapidRecipeSerializer(experiment)
+        data = serializer.data
+
+        fn = os.path.join(os.path.dirname(__file__), "experimentRecipe.json")
+
+        with open(fn, "r") as f:
+            json_schema = json.load(f)
+        self.assertIsNone(validate(instance=data, schema=json_schema))
```

Comment:
Oh right we had to do this becuase we couldn't validate with the shared repo last time.  Let's leave this like this and then we can file a separate ticket to switch to use the shared repo validator: https://github.com/mozilla/experimenter/issues/3058